lib/ansible/plugins/doc_fragments/qnos.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Peter Sprygada <psprygada@ansible.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+
+    # Standard files documentation fragment
+    DOCUMENTATION = r'''
+options:
```

Comment:
This file adds documentation for the provider dictionary which is used for local connection. For new platforms connection=local is not required hence this file can be removed

Reply:
Remove this file.